src/lib/effects.ts

```diff
@@ -1,14 +1,21 @@
 import { Effect, toSkill } from "kolmafia";
-import { $effects, have } from "libram";
+import {$effects, have, $familiar} from "libram";
+import {noncombatFamiliar} from "./familiar";
 
 function filterHave(effects: Effect[]) {
   return effects.filter((effect) => have(toSkill(effect)));
 }
 
 export function basicEffects(): Effect[] {
-  return filterHave(
-    $effects`Leash of Linguini, Empathy, Blood Bond, Springy Fusilli, Cletus's Canticle of Celerity`,
-  );
+  if(noncombatFamiliar() === $familiar`Frumious Bandersnatch`) {
```

Comment:
Can we make the list of affects an array, and append ode if we are using bander? The code will be clearer.